lldb/test/API/commands/statistics/basic/main.cpp

```diff
@@ -1,7 +1,13 @@
 // Test that the lldb command `statistics` works.
+#include <string>
+
+void foo() {
+  std::string str = "hello world";
```

Comment:
We want to test that this works for:
- built in C++ summary formatters (std::string)
- python summary formatters
- summary strings

As mentioned above, the same summary provider gets used for `std::string`, `std::string &`, `const std::string &`, `std::string *` and `const std::string *`. So add variants like this and make sure they all increment the one entry for `std::string`.
```
std::string str = "hello world";
std::string &str_ref = str;
const std::string &str_const_ref = str;
std::string *str_ptr = &str;
const std::string *str_const_ptr = &str;
```